src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const addFunction = function(number) {
+    if (number) {
```

Comment:
looks like prev solution was better.
this check not cover case when `number === 0`
it will work wiht some follow checks
`number !== undefined`
`number || number === 0`
`typeof number === 'number'`